clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -365,4 +365,142 @@ public void createServerGroup_errorIfCreateServiceFails()
         TASK_RETRY_SECONDS);
   }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given description w/ inputs, EC2 launch type "
+          + "with no load balancing successfully submit createServerGroup operation"
+          + "\n===")
+  @Test
+  public void createServerGroup_InputsEc2WithoutLoadBalacingTest()
+      throws IOException, InterruptedException {
+
+    // given
+    String url = getTestUrl(CREATE_SG_TEST_PATH);
+    String requestBody =
+        generateStringFromTestFile("/createServerGroup-inputs-ec2-withoutLoadBalacing.json");
+    String expectedServerGroupName = "ecs-integInputsEc2NoLoadBalancing";
+    setEcsAccountCreds();
+
+    // when
+    String taskId =
+        given()
+            .contentType(ContentType.JSON)
+            .body(requestBody)
+            .when()
+            .post(url)
+            .then()
+            .statusCode(200)
+            .contentType(ContentType.JSON)
+            .body("id", notNullValue())
+            .body("resourceUri", containsString("/task/"))
+            .extract()
+            .path("id");
+
+    retryUntilTrue(
+        () -> {
+          List<Object> taskHistory =
+              get(getTestUrl("/task/" + taskId))
+                  .then()
+                  .contentType(ContentType.JSON)
+                  .extract()
+                  .path("history");
+          if (taskHistory
+              .toString()
+              .contains(String.format("Done creating 1 of %s-v000", expectedServerGroupName))) {
+            return true;
+          }
+          return false;
+        },
+        String.format("Failed to detect service creation in %s seconds", TASK_RETRY_SECONDS),
+        TASK_RETRY_SECONDS);
+
+    // then
+    ArgumentCaptor<RegisterTaskDefinitionRequest> registerTaskDefArgs =
+        ArgumentCaptor.forClass(RegisterTaskDefinitionRequest.class);
+    verify(mockECS).registerTaskDefinition(registerTaskDefArgs.capture());
+    RegisterTaskDefinitionRequest seenTaskDefRequest = registerTaskDefArgs.getValue();
+    assertEquals(expectedServerGroupName, seenTaskDefRequest.getFamily());
+    assertEquals(1, seenTaskDefRequest.getContainerDefinitions().size());
+
+    ArgumentCaptor<CreateServiceRequest> createServiceArgs =
+        ArgumentCaptor.forClass(CreateServiceRequest.class);
+    verify(mockECS).createService(createServiceArgs.capture());
+    CreateServiceRequest seenCreateServRequest = createServiceArgs.getValue();
+    assertEquals("EC2", seenCreateServRequest.getLaunchType());
+    assertEquals(expectedServerGroupName + "-v000", seenCreateServRequest.getServiceName());
+  }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given description w/ inputs, EC2 launch type, legacy target group "
+          + "and service discovery registry fields, "
+          + "successfully submit createServerGroup operation"
+          + "\n===")
+  @Test
+  public void createServerGroup_InputsEc2LegacyTargetGroupServiceDiscoveryTest()
```

Comment:
since this test no longer uses target groups we should remove any mention of them from the `DisplayText`, test name, and test values (fields in operation json, server group name, etc.)

Reply:
Resolved.